800.renames-and-merges/e.yaml

```diff
@@ -67,6 +67,9 @@
 - { setname: enlightenment,            name: e17 }
 - { setname: enscript,                 namepat: "enscript-.*", ruleset: freebsd } # enscript-a4, enscript-letter, enscript-letterdj
 - { setname: entagged,                 name: entagged-tageditor, ruleset: gentoo }
+- { setname: environment-modules,      name: modules }
+- { setname: environment-modules,      name: env-modules }
+- { setname: environment-modules,      name: env-modules-tcl }
```

Comment:
Please add `addflavor: tcl` flag

Reply:
Understood. `env-modules-tcl` corresponds to the regular version 4 of Modules and `env-modules` to a compatibility version. So I have added a `compat` flavor to the `env-modules` name.

Reply:
Hmm, in this case flavor is not needed, I was misled by the package naming. Flavor is only needed to distinguish subpackages or variant packages, but not packages which only differ by version.

Reply:
Ok, so 1 rule to rename all 3 names to 'environment-modules'